File: CRZYSMKR.py
# CRZYSMKR - Crazy Smoker

# Author: Tarun Kumar
# E-mail: tarunkumar281200@gmail.com

from sys import stdin, stdout
# t=int(raw_input())
t=int(stdin.readline())
while(t>0):
    t=t-1
    # n=int(raw_input())
    n=int(stdin.readline())
    n=(11-((n%11)*8)%11)%11
    print n

File: CRZYSMKR.cpp
// CRZYSMKR - Crazy Smoker

// Author: Tarun Kumar
// E-mail: tarunkumar281200@gmail.com

#include <iostream>
using namespace std;

int main() {
		unsigned long long int t;
	cin>>t;
	while((t--)>0)
	{
	    unsigned long long n;
	    cin>>n;
	    cout<<(11-((n%11)*8)%11)%11<<endl;
	          
	}	
	return 0;
}
